### DISCUSSION TITLE
Wireless Adapter+8bitdo NeoGeo Update info\help

### DISCUSSION THREAD

**Discussion by Metro-City** [darthcloud/BlueRetro#1006] [Q&A]
Hi everyone, I recently got this wireless adapter+8bitdo pad, it says to update it choose the blueretro-neogeo.bin file but I can't find it in v24.04, which one do you use among these? Thank you

P.S. Does this happen to you too if you put 2 adapters on the neo geo CD and it doesn't recognize the second port and the screen remains grey? if I take it off it works.

**Comment by darthcloud** [comment]
File is called BlueRetro_hw1_parallel_1p.bin
